mapreader/load/images.py

```diff
@@ -114,36 +113,35 @@ def _resolve_file_path(file_path, file_ext):
         return files
 
     def __len__(self) -> int:
-        return int(len(self.images["parent"]) + len(self.images["patch"]))
+        return int(len(self.parents) + len(self.patches))
 
     def __str__(self) -> Literal[""]:
         print(f"#images: {self.__len__()}")
 
-        print(f"\n#parents: {len(self.images['parent'])}")
-        for i, img in enumerate(self.images["parent"]):
+        print(f"\n#parents: {len(self.parents)}")
+        for i, img in enumerate(self.parents):
             print(os.path.relpath(img))
             if i >= 10:
                 print("...")
                 break
 
-        print(f"\n#patches: {len(self.images['patch'])}")
-        for i, img in enumerate(self.images["patch"]):
+        print(f"\n#patches: {len(self.patches)}")
+        for i, img in enumerate(self.patches):
             print(os.path.relpath(img))
             if i >= 10:
                 print("...")
                 break
         return ""
 
-    def imagesConstructor(
+    def images_constructor(
```

Comment:
This feels like it should be a private method, hence `_images_constructor(...)`

Reply:
I also thought this but didnt get round to changing it